packages/trader/src/Modules/Trading/Containers/trade-chart.tsx

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import { ActiveSymbols } from '@deriv/api-types';
+import { getDecimalPlaces, isDesktop } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { useTraderStore } from 'Stores/useTraderStores';
+import { ChartBottomWidgets } from './chart-widgets';
+import SmartChartSwitcher from './smart-chart-switcher';
+import AccumulatorsChartElements from '../../SmartChart/Components/Markers/accumulators-chart-elements';
+import ToolbarWidgets from '../../SmartChart/Components/toolbar-widgets';
+import ToolbarWidgetsBeta from '../../SmartChartBeta/Components/toolbar-widgets';
+import AllMarkers from '../../SmartChart/Components/all-markers.jsx';
+import { TBottomWidgetsParams } from './trade';
+
+type TTradeChartProps = {
+    is_accumulator: boolean;
+    topWidgets: () => React.ReactElement;
+    bottomWidgets?: (props: TBottomWidgetsParams) => React.ReactElement;
+};
+
+const TradeChart = observer((props: TTradeChartProps) => {
+    const { is_accumulator, topWidgets } = props;
+    const { client, ui, common, contract_trade, portfolio } = useStore();
+    const {
+        accumulator_barriers_data,
+        accumulator_contract_barriers_data,
+        chart_type,
+        granularity,
+        markers_array,
+        has_crossed_accu_barriers,
+        updateGranularity,
+        updateChartType,
+    } = contract_trade;
+    const { all_positions } = portfolio;
+    const { is_chart_layout_default, is_chart_countdown_visible, is_dark_mode_on, is_mobile, is_positions_drawer_on } =
+        ui;
+    const { is_socket_opened, current_language } = common;
+    const { currency, is_beta_chart, should_show_eu_content } = client;
+    const {
+        chartStateChange,
+        is_trade_enabled,
+        main_barrier_flattened: main_barrier,
+        barriers_flattened: extra_barriers,
+        show_digits_stats,
+        symbol,
+        exportLayout,
+        setChartStatus,
+        chart_layout,
+        wsForget,
+        wsForgetStream,
+        wsSendRequest,
+        wsSubscribe,
+        active_symbols,
+        has_alternative_source,
+    } = useTraderStore();
+
+    const settings = {
+        countdown: is_chart_countdown_visible,
```

Comment:
`assetInformation` is not among settings expected by SmartCharts, hence removed `assetInformation: false` from this object.